Packs/ContentManagement/Scripts/CommitFiles/CommitFiles_test.py

```diff
@@ -23,6 +27,79 @@ def test_does_file_exist(mocker):
     assert not flag
 
 
+def test_does_file_exist_azure_devops(mocker):
+    """
+    Given:
+        - A branch name and a content file.
+    When:
+        - In order to know if this file already exists in the repository or not
+    Then:
+        - Returns True if the file exists and false if it doesn't
+    """
+    from CommitFiles import does_file_exist_azure_devops
+    mocker.patch.object(demisto, 'executeCommand', return_value=[{"Type": 1, "Contents": {}}])
+    flag = does_file_exist_azure_devops('demisto', content_file)
+    assert not flag
+
+
+def test_1_commit_content_item_azure_devops(mocker):
```

Comment:
rename to a name that represents what the test is: `test_commit_content_item_azure_devops_cant_find_branch`

Reply:
fixed